tests/Meilisearch.Tests/DocumentTests.cs

```diff
@@ -50,19 +69,37 @@ public async Task BasicDocumentsAdditionWithTimeoutError()
         [Fact]
         public async Task BasicDocumentsAdditionWithTimeoutErrorByInterval()
         {
-            var indexUID = "BasicDocumentsAdditionWithTimeoutError";
+            var indexUID = "BasicDocumentsAdditionWithTimeoutErrorByIntervalTest";
             Index index = await this.client.GetOrCreateIndex(indexUID);
 
             // Add the documents
             UpdateStatus update = await index.AddDocuments(new[] { new Movie { Id = "1", Name = "Batman" } });
             await Assert.ThrowsAsync<MeilisearchTimeoutError>(() => index.WaitForPendingUpdate(update.UpdateId, 0, 10));
         }
 
+        [Fact]
+        public async Task DocumentsAdditionWithPrimaryKey()
+        {
+            var indexUid = "DocumentsAdditionWithPrimaryKeyTest";
+            var index = this.client.Index(indexUid);
+            index.PrimaryKey.Should().BeNull();
+
+            // Add the documents
+            var document = await index.AddDocuments(new[] { new { movieId = "1", Name = "Batman" } }, "movieId");
```

Comment:
Use a primary key that does not contain `id`

Reply:
Done